lvmd/proto/lvmd.proto

```diff
@@ -40,6 +40,20 @@ message RemoveLVRequest {
     string device_class = 2;
 }
 
+message CreateLVSnapshotRequest {
+    string name = 1;              // The logical volume name.
+    repeated string tags = 2; // Tags to add to the volume during creation
+    string device_class = 3;
+    string sourcevolume = 4;      // Source lv of snap
+    uint64 size_gb = 5;           // Volume size in GiB.
+    string access_type = 6;       // Access type of snapshot
+    string snap_type    = 7;      // Snapshot type; can be thick or thin.
```

Comment:
it can be just `type` instead of `snap_type` as this required is all about snapshot?